triangle.py

```diff
@@ -0,0 +1,23 @@
+# Time Complexity : O(N) N -> Height of triangle
+# Space Complexity : Not sure ?? I just store the last row and keep updating the same 
```

Comment:
it would be O(n), since you are considering only the last row.

Reply:
Ok. Thanks!!